.github/workflows/docs.yml

```diff
@@ -0,0 +1,24 @@
+name: docs
+
+on: [push, pull_request]
```

Comment:
We should probably only execute this on pushes to the master branch of the original repo.

Reply:
changed to only running on master branch